canonical/canonical.com
# feat: /data/mongodb revamp

## Done

- Updated content and layout as per [doc](https://docs.google.com/document/d/1mCztOFZYbC3TeUM7j7pGfSoxKzBs20W1pJHZhdLqKs8/edit) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=13-18435&m=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1348.demos.haus/data/mongodb
- Check against doc and design
  - Don't forget to check smaller screens 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12347
